sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -329,6 +329,52 @@ def list_trials(self, name=None, namespace=None):
           {2} ".format(name, namespace, e))
         return result
 
+    def get_success_trial_details(self, name=None, namespace=None):
+        """
+        Get trial details that succeeded for an experiment
+        :param name: existing experiment name
+        :param namespace: defaults to current or default namespace
+        :return: trials name with hyperparams and metrics
+        """
+        if namespace is None:
+            namespace = utils.get_default_target_namespace()
+
+        thread = self.api_instance.list_namespaced_custom_object(
+            constants.KUBEFLOW_GROUP,
+            constants.KATIB_VERSION,
+            namespace=namespace,
+            plural=constants.TRIAL_PLURAL,
+            async_req=True)
+
+        katibtrial = None
+        try:
+            katibtrial = thread.get(constants.APISERVER_TIMEOUT)
+            result = []
+            for i in katibtrial.get("items"):
+                output = {}
+                if i.get("metadata", {}).get("ownerReferences")[0].get("name") == name:
+                    status = i.get("status", {}).get("conditions", [])[-1].get("type")
+                    if status == "Succeeded":
+                        output["name"] = i.get("metadata", {}).get("name")
+                        output["hyperparams"] = i.get("spec", {}).get("parameterAssignments", [])
```

Comment:
Maybe use here not short word, what do you think @Adarsh2910  ?
```suggestion
                        output["hyperparameters"] = i.get("spec", {}).get("parameterAssignments", [])
```

Reply:
yep, will fix it